internal/component/artifactory_release_source_test.go

```diff
@@ -88,6 +88,16 @@ var _ = Describe("interacting with BOSH releases on Artifactory", func() {
 				defer closeAndIgnoreError(f)
 				_, _ = io.Copy(res, f)
 			}), requireAuth))
+			artifactoryRouter.Handler(http.MethodGet, "/api/storage/basket/bosh-releases/smoothie/9.9/mango/mango-2.3.4-notices.zip", applyMiddleware(http.HandlerFunc(func(res http.ResponseWriter, _ *http.Request) {
+				res.WriteHeader(http.StatusOK)
+				// language=json
+				_, _ = io.WriteString(res, `{"checksums": {"sha1":  "some-sha-zip"}}`)
+			}), requireAuth))
+			artifactoryRouter.Handler(http.MethodGet, "/api/storage/basket/bosh-releases/smoothie/9.9/mango/mango-2.3.4-smoothie-9.9-notices.zip", applyMiddleware(http.HandlerFunc(func(res http.ResponseWriter, _ *http.Request) {
```

Comment:
Should this handler be responding to requests to get `/mango-2.3.4-smoothie-9.9-notices.tgz` to match the filename being returned on line 80?

Reply:
Thanks @davewalter, that's correct and I pushed a commit fixing this.